tests/utilities/test_argparse.py

```diff
@@ -168,3 +202,13 @@ def test_add_argparse_args_no_argument_group():
     args = parser.parse_args(fake_argv)
     assert args.main_arg == "abc"
     assert args.my_parameter == 2
+
+
+def test_gpus_arg_default():
+    assert _gpus_arg_default('1,2') == '1,2'
```

Comment:
shall this be list `[1,2]`?

Reply:
The argument has to be a string. Can change it to "[1,2]" if preferred, although the function just looks for a "," in the input.